code-of-conduct.md

```diff
@@ -0,0 +1,39 @@
+## Introduction
+
+Through participation in Moonlight Remote Work Encyclopedia, you will be interacting with other members of the Moonlight community, including both developers and clients. One of the core components of this is the Moonlight Community Slack Channel. These Community Guidelines and related procedures also apply to unacceptable behavior occurring outside the scope of community activities, in all community venues— online and in-person— as well as in all one-on-one communications, and anywhere such behavior has the potential to adversely affect the safety and well-being of community members.
+
+Your job engagements through Moonlight are covered by specific confidentiality requirements in the [Terms of Service](/terms). However, please treat all communications through Moonlight, including its Slack channel, as off-the-record and confidential information.
```

Comment:
Do we see striking up RWE conversations in slack?  Maybe the slack channel would be a good place to send contributors for discussion.